core/scripts/common/helpers.go

```diff
@@ -127,9 +128,9 @@ func SetupEnv(overrideNonce bool) Environment {
 		PanicErr(err)
 
 		owner.Nonce = big.NewInt(int64(nonce))
-		owner.GasPrice = gp.Mul(gp, big.NewInt(2))
 	}
-
+	owner.GasPrice = gp.Mul(gp, big.NewInt(2))
```

Comment:
so I ended up always having gas price multiplied by two, as sometimes it does not work with suggested price (basefee is bigger than maxfee error)


tried adding what you proposed and dont think it is reasonable to add it in this PR as it may require some fiddling
```
```

Reply:
Yeah, lets leave this for another PR.